app/components/file-details/scan-actions/manual-scan/index.ts

```diff
@@ -62,6 +63,29 @@ export default class FileDetailsScanActionsManualScanComponent extends Component
     ];
   }
 
+  manualScanStatus(scanStatus: [number]) {
+    const status = scanStatus[0];
+
+    const statusLabels = {
+      [ENUMS.MANUAL.NONE]: this.intl.t('notStarted'),
+      [ENUMS.MANUAL.REQUESTED]: this.intl.t('requested'),
+      [ENUMS.MANUAL.ASSESSING]: this.intl.t('inProgress'),
+      [ENUMS.MANUAL.DONE]: this.intl.t('completed'),
+    };
+
+    return statusLabels[status] || '';
+  }
```

Comment:
this can be just a getter
```
get manualScanStatusLabels() {
  return {
      [ENUMS.MANUAL.NONE]: this.intl.t('notStarted'),
      [ENUMS.MANUAL.REQUESTED]: this.intl.t('requested'),
      [ENUMS.MANUAL.ASSESSING]: this.intl.t('inProgress'),
      [ENUMS.MANUAL.DONE]: this.intl.t('completed'),
    };
}
```